doc/examples/debug/lib/main.dart

```diff
@@ -62,7 +62,7 @@ class MyGame extends BaseGame {
   final fpsTextConfig = TextConfig(color: const Color(0xFFFFFFFF));
 
   @override
-  bool debugMode() => true;
+  bool debugMode = true;
```

Comment:
Now that this can be set in the runtime, we could enhance this example, to have a button which enables and disables debug mode, what do you think?

Reply:
Sounds like a good idea, do you want to do it as a follow-up?

Reply:
👍